public/app/plugins/datasource/loki/modifyQuery.test.ts

```diff
@@ -38,7 +38,7 @@ describe('addLabelToQuery()', () => {
   });
 
   it('should work on arithmetical expressions', () => {
-    expect(addLabelToQuery('{} + {}', 'bar', '=', 'baz')).toBe('{bar="baz"} + {bar="baz"}');
+    expect(addLabelToQuery('{} + {}', 'bar', '=', 'baz')).toBe('{bar="baz"}+ {bar="baz"}');
```

Comment:
This seems like a bug somewhere. 

https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/loki/modifyQuery.ts#L30 was returning 
`streamSelectorPositions [ { from: 0, to: 2 }, { from: 5, to: 7 } ]` before. Now it is returning `streamSelectorPositions [ { from: 0, to: 3 }, { from: 5, to: 7 } ]`. I will have another look at that and will provide an update as soon as I know more. But for now, I don't feel we should just go ahead with this.

Reply:
[Oh, I did not read carefully.](https://github.com/grafana/grafana/pull/50866#pullrequestreview-1034135819)